### DISCUSSION TITLE
XIAO compatibility

### DISCUSSION THREAD

**Discussion by luni64** [luni64/EncoderTool#5] [Show and tell]
@m-r-m-s  and @FerGT50  
I improved the XIAO code (development branch) und tested it with the interrupt based encoders (`Encoder`) the polled encoder with / without buttons (`PolledEncoder`) and a 74165 based multiplexed setup (8 encoders with buttons, `EncPlex74165`). My tests show now problems so far. If you find time I'd appreciate some more testing.

![image](https://user-images.githubusercontent.com/12611497/119231699-7f107800-bb22-11eb-8d0b-6f96ec896ef9.png)

**Comment by m-r-m-s** [comment]
@luni64 sounds good and thank you for the update. It will be a little bit before I can test it out. I still need to get a Xiao dedicated bread board set up with my Din MIDI I/O circuit.

**Comment by FerGT50** [comment]
Sounds great! Thanks luni!
Will test with XIAO and WeMos D1 SAMD21 M0 Mini next weekend.
